cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControl.java

```diff
@@ -197,19 +199,20 @@ public static void sanityCheckCapacityEstimation(boolean allowCapacityEstimation
 
   /**
    * Add brokers
-   * @param brokerIds the broker ids.
-   * @param dryRun whether it is a dry run or not.
-   * @param throttleAddedBrokers whether throttle the brokers that are being added.
-   * @param goals the goals to be met when adding the brokers. When empty all goals will be used.
+   * @param brokerIds The broker ids.
+   * @param dryRun Whether it is a dry run or not.
+   * @param throttleAddedBrokers Whether throttle the brokers that are being added.
+   * @param goals The goals to be met when adding the brokers. When empty all goals will be used.
    * @param requirements The cluster model completeness requirements.
    * @param operationProgress The progress of the job to update.
    * @param allowCapacityEstimation Allow capacity estimation in cluster model if the requested broker capacity is unavailable.
    * @param concurrentPartitionMovements The maximum number of concurrent partition movements per broker
    *                                     (if null, use num.concurrent.partition.movements.per.broker).
    * @param concurrentLeaderMovements The maximum number of concurrent leader movements
    *                                  (if null, use num.concurrent.leader.movements).
+   * @param skipHardGoalCheck Whether check and add all hard goals to {@code goals}.
```

Comment:
If we rephrase it as follows, would it be more clear?
`Whether check and add all hard goals to {@code goals}.` -> `True if the provided {@code goals} do not have to contain all hard goals, false otherwise`